fmt.go

```diff
@@ -0,0 +1,76 @@
+package clues
```

Comment:
moved here from datanode.go.  The only additional change is the addition of the exported Fmt func, which is consumed by Clog.